osd/user-ca-bundle_invalid.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Info",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Action required: Misconfigured additional Trusted CA bundle",
+    "description": "Your cluster requires you to take action because the Additional Trusted CA Bundle that you have applied to your cluster is corrupt or cannot be correctly parsed. This may impact your cluster's ability to communicate with any external resources that require that bundle, including those accessible via a cluster-wide proxy. Please remove or update the Additional Trusted CA Bundle to ensure it is in a parseable, plaintext PEM-formatted format. If you require any further assistance, please file a support request.",
```

Comment:
will we consider the expired CA as invalid as well?

Reply:
For that situation we have https://github.com/openshift/managed-notifications/blob/master/osd/cluster-wide_proxy_ca_expiring.json  which is a bit more specifically written. 
Our "ClusterProxyCAExpiringSRE" alert SOP directs SREs towards using that template for when that happens.